connectors/src/connectors/zendesk/lib/zendesk_api.ts

```diff
@@ -181,25 +182,24 @@ async function fetchFromZendeskWithRetries({
 /**
  * Fetches a batch of categories from the Zendesk API.
  */
-export async function fetchZendeskCategoriesInBrand({
-  brandSubdomain,
-  accessToken,
-  pageSize,
-  cursor = null,
-}: {
-  brandSubdomain: string;
-  accessToken: string;
-  pageSize: number;
-  cursor: string | null;
-}): Promise<{
+export async function fetchZendeskCategoriesInBrand(
+  accessToken: string,
+  {
+    brandSubdomain,
+    pageSize,
+    url,
+  }:
+    | { brandSubdomain: string; pageSize: number; url?: never }
+    | { brandSubdomain?: never; pageSize?: never; url: string }
+): Promise<{
   categories: ZendeskFetchedCategory[];
-  meta: { has_more: boolean; after_cursor: string };
+  meta: { has_more: boolean };
```

Comment:
aren't we missing `after_cursor` here ?
nit: since this is our code, should we use camel case so the snake case stuff is contained inside this file ?

Reply:
the fallback value was actually incorrect, we want `links` everywhere, fixed now

Reply:
turning everything in camelCase